packages/lexical-editor/src/components/Toolbar/Toolbar.tsx

```diff
@@ -30,15 +27,49 @@ interface FloatingToolbarProps {
 
 const FloatingToolbar: FC<FloatingToolbarProps> = ({ children, type, anchorElem, editor }) => {
     const popupCharStylesEditorRef = useRef<HTMLDivElement | null>(null);
-    const { toolbarType, setToolbarType, setTextBlockSelection } = useRichTextEditor();
-    const [activeEditor, setActiveEditor] = useState(editor);
+    const { toolbarType, setToolbarType } = useRichTextEditor();
 
     useEffect(() => {
         if (toolbarType !== type) {
             setToolbarType(type);
         }
     }, [type]);
 
+    function mouseMoveListener(e: MouseEvent) {
```

Comment:
Can we `useCallback` here? If possible, let's avoid recreating the same functions on each render.